### ISSUE TITLE
Goals?

### ISSUE THREAD

**Issue by dsernst** [beeminder/chromeminder#1]
Also curious what you're interested in accomplishing with Chromeminder. Easy access to 'My Goals' could be handy.

**Comment by dreeves** [comment]
Agreed. And more in-your-face notifications of beemergencies might be
pretty powerful.

On Thu, Mar 6, 2014 at 2:58 AM, David Ernst notifications@github.comwrote:

> Also curious what you're interested in accomplishing with Chromeminder.
> Easy access to 'My Goals' could be handy.
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/beeminder/chromeminder/issues/1
> .

## 

http://dreev.es  --  search://"Daniel Reeves"
Goal tracking + Commitment contracts == http://beeminder.com
